src/main/java/mate/jdbc/service/DriverServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Driver get(Long id) {
+        return driverDao.get(id).orElseThrow(
+                () -> new IllegalArgumentException("Driver with id " + id + " not found"));
```

Comment:
```suggestion
        return driverDao.get(id).get();
```


I think `NoSuchElementException` more suitable here. (IllegalArgumentException thrown to indicate that a method has been passed an illegal or inappropriate argument. In this case we pass legal argument, but Driver with this id does not exist)